helm/charts/kratos/values.yaml

```diff
@@ -425,6 +429,10 @@ statefulSet:
   #      lines, adjust them as necessary, and remove the curly braces after 'labels:'.
   #      e.g.  type: app
 
+  # -- Pod priority
+  # https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  # priorityClassName: ""
```

Comment:
```suggestion
  priorityClassName: ""
```

Reply:
empty string is evaluated to false in both if and with clauses, so we can safely use it as a default value :)